src/tests/ftest/util/dmg_utils.py

```diff
@@ -411,8 +414,14 @@ def pool_create(self, scm_size, uid=None, gid=None, nvme_size=None,
             "scm_size": scm_size,
             "nvme_size": nvme_size,
             "nsvc": svcn,
-            "acl_file": acl_file
+            "acl_file": acl_file,
+            "nranks": nranks,
+            "size": size
         }
+
+        if not size and not scm_size:
+            return False
```

Comment:
```suggestion
```
I don't think you need these lines. If you don't set either, dmg will fail and I think it'll throw a CommandFailure.

[root@wolf-3 ~]# dmg pool create
ERROR: dmg: either --size or --scm-size must be supplied

Reply:
Will remove, as Phil mentioned in another comment this may be useful for negative testing.